packages/map-template/src/components/Search/Categories/Categories.scss

```diff
@@ -3,16 +3,21 @@
     flex-shrink: 0;
     display: grid;
     grid-auto-flow: column;
-    align-items: end;
     gap: var(--spacing-x-small);
+    margin-top: var(--spacing-x-small);
 
+    ::-webkit-scrollbar {
```

Comment:
Will this work in Firefox and Safari?

Reply:
I may just answer this myself: It does not work in Firefox 😄

Reply:
Can you send me a screenshot of how it looks on Firefox please? 🙏🏻

Reply:
No, because you fixed it 😄